src/ngx_http_lua_timer.c

```diff
@@ -568,6 +575,9 @@ ngx_http_lua_timer_handler(ngx_event_t *ev)
 
     c = ngx_http_lua_create_fake_connection(tctx.pool);
     if (c == NULL) {
+        ngx_log_error(NGX_LOG_ALERT, ngx_cycle->log, 0,
```

Comment:
Better use the "error" level instead of "alert"?

Reply:
Just follow the level of the first two error message when timer failed to run.